app/views/claims/schools/claims/mentor_trainings/edit.html.erb

```diff
@@ -24,29 +24,29 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <span class="govuk-caption-l"><%= t(".add_claim", school: @claim.provider.name) %></span>
+        <h2 class="govuk-heading-l"><%= t(".hours_of_training_for_mentor", mentor: mentor_training_form.mentor_full_name) %> </h2>
+
+        <% if !mentor_training_form.max_hours_equals_maximum_claimable_hours? %>
+          <%= govuk_inset_text text: sanitize(t(".disclaimer", count: mentor_training_form.max_hours, mentor_name: f.object.mentor_full_name, provider_name: f.object.provider_name, support_link: govuk_link_to("ittmentor.funding@education.gov.uk", "#"))) %>
+        <% end %>
```

Comment:
In other words, what about changing this?

So the `text:` param you pass in to `govuk_inset_text` becomes something more like:

```ruby
<%= govuk_inset_text text: (
  tag.p(t(".claimable_hours_left", count: mentor_training_form.max_hours, mentor_name: f.object.mentor_full_name, provider_name: f.object.provider_name)) +
  tag.p(t(".contact_support", support_link: govuk_link_to("ittmentor.funding@education.gov.uk", "#")))
) %>
```

Which would result in:

```html
<div class="govuk-inset-text">
  <p>There are %{count} hours left to claim for %{mentor_name} for %{provider_name}.</p>
  <p>Contact %{support_link} if you think there is a problem.</p>
</div>
```

Doing this means there's no need for the CSS override ✨🧹

Reply:
I'd also take this one step further and suggest that this is probably a good candidate for conversion into a ViewComponent.

- It contains logic to decide whether it shows (e.g. `if !mentor_training_form.max_hours_equals_maximum_claimable_hours?`) which would be a great use of ViewComponent [conditional rendering](https://viewcomponent.org/guide/conditional_rendering.html).
- It seems quite 'clever' for a view. It passes numerous params to the translation, so it has knowledge of which values go where. I think that could be structured more cleanly as helper methods in a ViewComponent.
- The component and surrounding logic has already been duplicated into [two](https://github.com/DFE-Digital/itt-mentor-services/blob/0f0bd1b4a1b734c527b205e4494c4cbabd9b8100/app/views/claims/schools/claims/mentor_trainings/edit.html.erb#L29-L31) [places](https://github.com/DFE-Digital/itt-mentor-services/blob/0f0bd1b4a1b734c527b205e4494c4cbabd9b8100/app/views/claims/support/schools/claims/mentor_trainings/edit.html.erb#L29-L31) 🙂 so it would DRY things up.